radio/radio_comm.py

```diff
@@ -47,165 +47,27 @@ def in_prompt_tokens(self, cli=None):
     sender = Sender(args.uplink_host, args.uplink_port)
     rcv = Receiver(args.downlink_host, args.downlink_port)
 
-    def receive():
-        data = rcv.decode_kiss(rcv.receive())
-        return frame_decoder.decode(data)
-
-    def receive_raw():
-        return rcv.decode_kiss(rcv.receive())
-
-    def set_timeout(timeout_in_ms=-1):
-        rcv.timeout(timeout_in_ms)
-
-    def send(frame):
-        sender.send(frame)
-
-    def send_receive(frame):
-        send(frame)
-        return receive()
-
-    def get_sender():
-        return sender
-
-    def get_receiver():
-        return rcv
-
-    def get_beacon():
-        from tools.parse_beacon import ParseBeacon
-        return ParseBeacon.parse(send_receive(SendBeacon()))
-
     def get_file(file_dict):

```

Comment:
To jest jakoś używane w obecnym systemie?

Reply:
Nie mam pojęcia :) Wydaje mi się, że nie, ale zostawiłem bo moze komuś się przypomni. Jak nie, to wywalę po standupie